BRAINSCommonLib/Slicer3LandmarkIO.cxx

```diff
@@ -230,7 +230,7 @@ ReadSlicer3toITKLmkOldSlicer(const std::string & landmarksFilename)
     throw itk::ImageFileReaderException(__FILE__, __LINE__, errorMsg.c_str(), ITK_LOCATION);
     // do not return empty landmarks
   }
-  bool useLPS = false;
+  bool        useLPS = false;
```

Comment:
This commit has 2 changes in it. 
1) formatting changes
2) changes related to the commit message.

Please  separate the changes.  Make all the clang-format changes in one commit for the entire BRAINSTools tree, then namek other changes.